dags/solutions_team/configs/tensorflow/common.py

```diff
@@ -81,7 +81,7 @@ def install_tf_2_16() -> Tuple[str]:
   """Install tf 2.16 + libtpu."""
   return (
       "pip install tensorflow-text-nightly",
-      "sudo gsutil -m cp gs://cloud-tpu-v2-images-dev-artifacts/tensorflow/2.16/2024-02-20/*.whl /tmp/ && pip install /tmp/tf*.whl --force",
+      "sudo gsutil -m cp gs://cloud-tpu-v2-images-dev-artifacts/tensorflow/2.16/2024-02-20/t*.whl /tmp/ && pip install /tmp/t*.whl --force",
```

Comment:
Instead of t*.whl, can we make it tensorflow*.whl

Reply:
My reasoning for `t*` was that sometimes I'll see the `tf_nightly*.whl` pattern and sometimes I'll see the`tensorflow*.whl` pattern so I figured this would be a little more resilient.

I'm not strongly opposed to `tensorflow*.whl` and can make that change if you feel strongly about it.